File: app/src/main/java/org/dhis2/utils/jira/JiraIssue.java
package org.dhis2.utils.jira;

/**
 * QUADRAM. Created by ppajuelo on 11/04/2019.
 */
public class JiraIssue {
    private int id;
    private String key;
    private JiraIssueField fields;

    public int getId() {
        return id;
    }

    public String getKey() {
        return key;
    }

    public JiraIssueField getFields() {
        return fields;
    }
}

File: app/src/main/java/org/dhis2/data/jira/JiraIssue.kt
package org.dhis2.data.jira

data class JiraIssue(
    val id: Int = 0,
    val key: String,
    val fields: JiraIssueField? = null,
)

data class JiraIssueField(
    val issuetype: JiraField? = null,
    var summary: String? = null,
    val status: JiraField? = null,
)

data class JiraField(
    val name: String? = null,
    val id: Int = 0,
)
